src/main/scala/gen/gmos/GmosNLongslitD.scala

```diff
@@ -270,53 +270,49 @@ object GmosNLongslitD {
   }
 
   def apply[F[_]: Sync : Timer](
-    mode:      ObservingMode.Spectroscopy.GmosNorth,
-    magnitude: MagnitudeValue
+    itc:           Itc[F],
+    targetProfile: TargetProfile,
+    observingMode: ObservingMode.Spectroscopy.GmosNorth,
+    signalToNoise: Int
   ): GmosNLongslitD[F] =
 
     new GmosNLongslitD[F] with GmosNOps with GmosLongslitMath {
 
       val emptySequence: Stream[F, Step.GmosN] =
         Stream.empty.covaryAll[F, Step.GmosN]
 
-      // This is a placeholder for the integration time lookup.  It will
-      // surely need other parameters for current conditions and to
-      // distinguish acquisition from science.
-      def acquisitionTime(itc: ImaginaryItc[F]): F[FiniteDuration] =
-        itc.integrationTime(mode, magnitude)
+      // This is a placeholder for the integration time lookup.
+      def acquisitionTime: F[FiniteDuration] =
+        Sync[F].delay { FiniteDuration(10, SECONDS) }
```

Comment:
I don't know how to call the `Itc` to get the acquisition time yet, so this is a stand-in.

Reply:
I have been doing `import scala.concurrent.duration._` which lets you say `10.seconds`, just FYI.

Reply:
The `GmosCommonDynamicConfig` records exposure times using a Java `Duration`, but I think we will want to change that to `FiniteDuration` to be consistent.  I didn't use the `10.seconds` syntax here though because I'd added similar syntax already for the Java `Duration`.  That can be fixed when exposure times are using `FiniteDuration`.